src/include/conv_fin.hpp

```diff
@@ -1129,22 +1096,21 @@ int ConvFin<Tgpu, Tref>::MIOpenFind()
                                                       workspace.gpuData.buf.get(),
                                                       workspace.desc.GetNumBytes(),
                                                       convDesc.attribute.gfx90aFp16alt.GetWrW()};
-                    for(auto idx = 0; idx < INVOKE_LIMIT; idx++)
-                        invoker(h, invoke_ctx);
+                    kernel_time = BenchmarkInvoker(invoker, h, invoke_ctx);
                 }
                 else
                 {
                     throw std::runtime_error("Invalid Direction");
                 }
+
+                res_item["time"]   = kernel_time;
+                res_item["reason"] = "Success";
```

Comment:
for argument sake, what if Benchmark invoker return k_time=0? is the reason still success?

Reply:
Maybe something like: 'Could not retrieve ktime from MIOpen' so we know where ktime=0 comes from, since we have seen this before

Reply:
Added a print for ktime == 0